docs/en/glossary/glossary.asciidoc

```diff
@@ -25,6 +25,12 @@ ZooKeeper to {es}.
 include::{kib-repo-dir}/glossary.asciidoc[tag=advanced-settings-def]
 --
 
+[[glossary-agent-policy]] Agent policy::
+A collection of inputs and settings that defines the data to be collected by {agent}.
```

Comment:
An agent policy can be applied to a single agent or shared by a group of agents. This makes it easier to manage many agents at scale.